src/apps/cif2bcif/converter.ts

```diff
@@ -4,20 +4,45 @@
  * @author David Sehnal <david.sehnal@gmail.com>
```

Comment:
don't forget to add you name to the authors of a file if you contributed substantially

Reply:
Fair enough. Normally, I don't do that as the git history/blame points to authors reliably. Added in 9b85008